docs/editor_manual/finding_your_way_around/the_explorer_page.md

```diff
@@ -4,31 +4,22 @@ The Explorer page allows you to view a page's children and perform actions on th
 
 ![](../../_static/images/screen05_explorer_page.png)
 
--   The name of the section you are looking at is displayed below the breadcrumb (the row of page names beginning with the home icon). Each section is also itself a page (in this case the homepage). Clicking the title of the section takes you to the Edit screen for the section page.
+-   The name of the section you are looking at is displayed at the top. Each section is also itself a page (in this case the homepage). Clicking the title of the section takes you to the Edit screen for the section page.
 -   As the heading suggests, below are the child pages of the section. Clicking the titles of each child page will take you to its Edit screen.
-
-![](../../_static/images/screen06_explorer_page_arrow.png)
-
 -   Clicking the arrows will display a further level of child pages.
 
 ![](../../_static/images/screen07_explorer_page_breadcrumb.png)
 
--   As you drill down through the site the breadcrumb (the row of pages beginning with the home icon) will display the path you have taken. Clicking on the page titles in the breadcrumb will take you to the Explorer screen for that page.
-
-![](../../_static/images/screen08_add_page_button.png)
```

Comment:
Removed because this is now redundant with the "page_more_dropdown" screenshot below.